KsApi/models/PostCommentEnvelope.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+public struct PostCommentEnvelope: Decodable {
```

Comment:
I guess this doesn't really require a test because it's pretty straightforward wrapper.